recipes/openexr/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "openexr" %}
+{% set version = "2.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openexr/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 8243b7de12b52239fe9235a6aeb4e35ead2247833e4fbc41541774b222717933
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+    - {{ pin_compatible('python', lower_bound='2.7') }}
```

Comment:
What's the `pin_compatible` here for? If the goal is to just ensure `python` is in the build environment and is at least `2.7`, then listing `python` should be sufficient. If you want to force Python `2.7`, then you can add `python 2.7.*`.